docs/Rollup_basics.md

```diff
@@ -13,42 +13,45 @@ and lower cost than on-chain Ethereum contracts.
 Despite running off-chain, ArbChains are _trustless_, meaning that any
 party can force an ArbChain to execute correctly according to the code of its contracts.
 
-You can make as many ArbChains as you want.
-Typically, you'll want one ArbChain for each dapp you want to create.
+You can make as many ArbChains as you want. Each ArbChain provides the same interface as the Ethereum blockchain, and contracts can be deployed on the chain at anytime using the `CREATE` and `CREATE2` opcode.
```

Comment:
Should say that you can also deploy contracts by sending transactions to address zero on the chain, just as you would on Ethereum.